vm-image-spec.yaml

```diff
@@ -353,6 +358,18 @@ merge: |
       && echo 'root - nofile 1048576' >>/etc/security/limits.conf \
          )
 
+  # Allow postgres user (compute_ctl) to run swap resizer.
+  # Need to install sudo in order to allow this.
+  #
+  # Also, remove the 'read' permission from group/other on /neonvm/bin/resize-swap, just to be safe.
+  RUN set -e \
+      && apt update \
+      && apt install --no-install-recommends -y \
+             sudo \
+      && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
+      && chmod 711 /neonvm/bin/resize-swap
```

Comment:
Hm, this doesn't work because the neonvm-internal stuff is added after any "user-defined" parts run.

So this actually needs to run on VM startup AFAICT

Reply:
9a373a1e1ff6fc78ef6b70c0ce1686a78431d0d1